src/components/NewCommentForm.tsx

```diff
@@ -14,24 +82,35 @@ export const NewCommentForm: React.FC = () => {
             name="name"
             id="comment-author-name"
             placeholder="Name Surname"
-            className="input is-danger"
+            className={classNames('input', {
+              'is-danger': isNameError,
+            })}
+            value={name}
+            onChange={(e) => {
+              setName(e.target.value);
+              setIsNameError(false);
+            }}
```

Comment:
Do not use anonymous function use separate handler
It will works for all fields
```
const onFieldChange = ({taget: {value, name}}) => {
  setForm(prevState => {...prevState, [name]: value })
} 
```